src/test/scala/mesosphere/marathon/api/v2/AppsResourceTest.scala

```diff
@@ -356,9 +356,15 @@ class AppsResourceTest extends MarathonSpec with MarathonActorSupport with Match
           ))
         ))
       )),
-      portDefinitions = Nil
+      portDefinitions = Seq.empty
     )
-    val (body, plan) = prepareApp(app)
+
+    val group = Group(PathId("/"), Set(app))
+    val plan = DeploymentPlan(group, group)
+    val body = Json.stringify(Json.toJson(app).as[JsObject] - "ports").getBytes("UTF-8")
```

Comment:
pretty silly that we generate invalid JSON, then correct with some JSON algebra, in order to process it effectively. We **really** shouldn't be generating JSON that doesn't pass our own validation routines. very sad to see this